lib/redcord/vacuum_helper.rb

```diff
@@ -31,18 +31,32 @@ def self._vacuum_range_index_attribute(model, range_index_attr)
 
     # Handle nil values for range index attributes, which are stored in a normal
     # set at Redcord:Model:range_index_attr:
-    model.redis.scan_each_shard("#{range_index_set_nil_key}*") do |key|
+    model.redis.scan_each_shard(range_index_set_nil_key) do |key|
```

Comment:
Would this and line 42 below be a no-op for when it is in cluster mode?

Could we add a quick comment explaining the difference here between what line 34 and 38 do?

Reply:
oh yeah! let's check and see if cluster mode is enabled here for this model. That'd be clearer